src/RgbdCameraSensor.cc

```diff
@@ -390,6 +381,15 @@ bool RgbdCameraSensor::CreateCameras()
         std::placeholders::_1, std::placeholders::_2, std::placeholders::_3,
         std::placeholders::_4, std::placeholders::_5));
 
+  // Initialize the point message.
+  // \todo(anyone) The true value in the following function call forces
+  // the xyz and rgb fields to be aligned to memory boundaries. This is need
+  // by ROS1: https://github.com/ros/common_msgs/pull/77. Ideally, memory
+  // alignment should be configured.
+  msgs::InitPointCloudPacked(this->dataPtr->pointMsg, this->OpticalFrameId(), true,
+      {{"xyz", msgs::PointCloudPacked::Field::FLOAT32},
+       {"rgb", msgs::PointCloudPacked::Field::FLOAT32}});
```

Comment:
CI picked up a minor cpplint issue here:

```
/github/workspace/src/RgbdCameraSensor.cc:389:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
```

```suggestion
  msgs::InitPointCloudPacked(this->dataPtr->pointMsg, this->OpticalFrameId(), 
      true, 
      {{"xyz", msgs::PointCloudPacked::Field::FLOAT32},
       {"rgb", msgs::PointCloudPacked::Field::FLOAT32}});
```

Reply:
Fixed in new commit :)